commit c0ad6bb378d05ae877d87b9192bdc921088e6d84 (refs/changes/10/2310/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-07-31T16:31:19-07:00 (2 years, 2 months ago)
    
    joyent/node-triton#219 test: Fix test suite to not use non-existant `triton img rm -w` wait option

diff --git a/test/integration/helpers.js b/test/integration/helpers.js
index 187ce2a..e091dda 100644
--- a/test/integration/helpers.js
+++ b/test/integration/helpers.js
@@ -468,7 +468,7 @@ function deleteTestImg(t, imgNameOrId, cb) {
             }
         } else {
             var imgToRm = JSON.parse(stdout);
-            safeTriton(t, ['img', 'rm', '-w', imgToRm.id], function onRm() {
+            safeTriton(t, ['img', 'rm', '-f', imgToRm.id], function onRm() {
                 t.ok(true, 'deleted img ' + imgToRm.id);
                 cb();
             });
